clusterman/draining/queue.py

```diff
@@ -395,7 +395,11 @@ def terminate_host(host: Host) -> None:
     logger.info(f'Terminating: {host.instance_id}')
     resource_group_class = RESOURCE_GROUPS[host.sender]
     resource_group = resource_group_class(host.group_id)
-    resource_group.terminate_instances_by_id([host.instance_id])
+
+    try:
+        resource_group.terminate_instances_by_id([host.instance_id])
+    except Exception:
+        logger.error(f'Failed to terminate host: {host.hostname}, possibly already terminated.')
```

Comment:
nit: include the actual error text instead of printing an assumption of what went wrong?